views.py

```diff
@@ -167,6 +167,14 @@ def page_detail(request, page_id, book_id=None):
             context['date']=book_json['dateCreated'][0:4]
         except:
             context['date']="n.d."
+    try:
+        context['note'] = book_json['note']
```

Comment:
I think this line can be removed, because context['note'] will be set to something else regardless of what happens.